plasma_framework/test/src/exits/payment/PaymentInFlightExitable.test.js

```diff
@@ -32,26 +32,36 @@ contract('PaymentInFlightExitable', ([_, alice, bob, carol]) => {
     const INCLUSION_PROOF_LENGTH_IN_BYTES = 512;
     const IN_FLIGHT_TX_WITNESS_BYTES = web3.utils.bytesToHex('a'.repeat(WITNESS_LENGTH_IN_BYTES));
     const BLOCK_NUMBER = 1000;
+    const DEPOSIT_BLOCK_NUMBER = BLOCK_NUMBER + 1;
     const DUMMY_INPUT_1 = '0x0000000000000000000000000000000000000000000000000000000000000001';
     const DUMMY_INPUT_2 = '0x0000000000000000000000000000000000000000000000000000000000000002';
     const MERKLE_TREE_HEIGHT = 3;
     const AMOUNT = 10;
     const TOLERANCE_SECONDS = new BN(1);
 
     describe('startInFlightExit', () => {
-        function buildValidIfeStartArgs(amount, [ifeOwner, inputOwner1, inputOwner2], blockNum) {
-            // TODO: make one of those a deposit transaction
+        function buildValidIfeStartArgs(amount, [ifeOwner, inputOwner1, inputOwner2], blockNum1, blockNum2) {
             const inputTx1 = createInputTransaction(DUMMY_INPUT_1, inputOwner1, amount);
-            const inputTx2 = createInputTransaction(DUMMY_INPUT_2, inputOwner2, amount);
+            const inputTx2 = createDepositTransaction(inputOwner2, amount);
```

Comment:
seems like the `blockNum1/2` does not need to be normal tx or deposit tx. Perhaps do a if-else here for both inputTx?
```
const inputTx = isDeposit(blockNum)? createDepositTransaction(...) : createInputTransaction(...);
```